devbuild.sh

```diff
@@ -237,6 +243,20 @@ if [ "${VERBOSE}" = true ] ; then
   settings
 fi
 
+# Check out external components ===============================================
+if [ "${APPLICATION}" = "ATMAQ" ]; then
+  if [ "${DEFAULT_BUILD}" = true ]; then
+    BUILD_NEXUS="on"
+    BUILD_AQM_UTILS="on"
+    BUILD_UPP="off"
+  fi
+  if [ "${PLATFORM}" = "wcoss2" ]; then
+    BUILD_POST_STAT="on"
+  else
+    BUILD_POST_STAT="off"
+  fi
+fi
```

Comment:
It would appear that this section would be better placed around line 203 and the `# Check out external components` should be renamed to:

`# Choose components to build for Online-CMAQ`

since the use of manage_externals is now done outside of `devbuild.sh`.

Reply:
@MichaelLueken, since this if-statement uses `PLATFORM`, I've moved the `DEFAULT_BUILD` if-statement together.